modules/oe_webtools_search_widget/src/Plugin/Block/SearchWidgetBlock.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\oe_webtools_search_widget\Plugin\Block;
+
+use Drupal\Component\Serialization\Json;
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Session\AccountInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides Webtools Widget Search block.
+ *
+ * @Block(
+ *   id = "oe_webtools_search_widget",
+ *   admin_label = @Translation("OpenEuropa Webtools Search widget"),
+ *   category = @Translation("Forms"),
+ * )
+ */
+class SearchWidgetBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function blockAccess(AccountInterface $account) {
+    return AccessResult::allowedIfHasPermission($account, 'access content');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
```

Comment:
Just a suggestion, we could add `defaultConfiguration` with default empty for providing a form selector to be used in the build with the webtools json, since they allow this now.